_posts/2022-01-06-hhvm-4.143.markdown

```diff
@@ -0,0 +1,31 @@
+---
+title: "HHVM 4.143"
+layout: post
+author: fred
+category: blog
+---
+
+HHVM 4.143 is released! This release marks the end of support of 4.135;
+4.136&ndash;4.140 remain supported, as do th 4.102 and 4.128 LTS releases.
+
+Additional patch releases have been made for 4.102, 4.128, and 4.135-4.140,
+which change the typechecker to ignore any files in `vendor/bin/`; this is
+required for compatibility with Composer 2.2.
+
+# Highlights
+
+- Compatible with Composer 2.2.
+- Added documentation on hover for `readonly` and `extends` keywords.
```

Comment:
I believe that the commit that adds hover info for `async` and friends is also in 4.143. The commit that added this was made two commits after the vendor/bin commit.
```suggestion
- Added documentation on hover for `async` `async {...}`, `await`, `concurrent`, `extends`, and `readonly`.
```

https://github.com/facebook/hhvm/commit/e2be871af1b3ea6ac252909380569fa5d0593338